src/pages/cli/cm/upload.md

```
@@ -0,0 +1,87 @@
+│   ├── collection.png      # Collection image
+│   └── collection.json     # Collection metadata
+└── asset-cache.json        # Generated after upload
+```
+
+## Upload Process
+
+1. Asset Discovery: The command automatically scans the `assets/` directory and identifies image, metadata and collection files.
+2. Validation Phase: Verifies the integrity of your files, for example if all image files have matching metadata files and that the metadata is valid json.
+3. Cache Check: To identify which files were already uploaded the `asset-cache.json` file is validated.
+4. Upload: The actual upload is done.
+5. Cache Generation: The `asset-cache.json` file is generated 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)